Source/Urho3D/Audio/SoundSource.cpp

```diff
@@ -403,6 +403,7 @@ void SoundSource::Mix(int dest[], unsigned samples, int mixRate, SpeakerMode mod
         {
             switch (mode)
             {
+            case SPK_AUTO: break;
```

Comment:
Can we actually get to this if legally? Consider doing URHO3D_ASSERT, maybe together with default: label